include/cufinufft/types.h

```diff
@@ -0,0 +1,169 @@
+#ifndef CUFINUFFT_TYPES_H
+#define CUFINUFFT_TYPES_H
+
+#include <cufft.h>
+
+#include <cufinufft_opts.h>
+#include <finufft_spread_opts.h>
+
+#include <complex>
+
+#define CUFINUFFT_BIGINT int
```

Comment:
Is this a temp fix, or should we move to int64_t as in CPU code?   Maybe you are just in-progress here...